src/sentry/models/grouptype.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+from sentry.db.models import BoundedPositiveIntegerField, Model, region_silo_only_model
+from sentry.types.issues import GroupCategory
+
+
+@region_silo_only_model
+class GroupType(Model):
+    __include_in_export__ = False
+
+    slug = models.CharField(max_length=254, unique=True)
+    description = models.CharField(max_length=254, unique=True)
+    category = BoundedPositiveIntegerField(
+        choices=[(category.value, category.name.lower()) for category in GroupCategory],
```

Comment:
Is this safe to do? I saw a similar usage in the `GroupTombstone` table's `level` column.

Reply:
This defines column metadata that django can use to do validation. It looks like your goals include dynamic category values that aren't defined in application code, which this wouldn't accommodate.